templates/remix-javascript/vite.config.js

```diff
@@ -5,5 +5,13 @@ import { defineConfig } from "vite";
 installGlobals();
```

Comment:
Do we want `nativeFetch:true` here too?

Reply:
🤦 

Yep! Thanks for pointing out what I already told you I'd add 😅

Reply:
Actually, I'm just going to bump all the templates to node 20 and remove `installGlobals` since we are going to deprecate it once node 18 is no longer LTS

Thanks!